brayns/io/MeshLoader.cpp

```diff
@@ -33,6 +33,9 @@
 
 #include <brayns/common/scene/Scene.h>
 
+#define BRAYNS_USE_ASSIMP \
+    1 //************* REMOVE BEFORE PUSH ******************
```

Comment:
??

Reply:
Comment was obliviously not big enough... :see_no_evil: